doc/whats_new/v0.24.rst

```diff
@@ -61,6 +61,15 @@ Changelog
   change since `None` was defaulting to these values already.
   :pr:`16493` by :user:`Darshan N <DarshanGowda0>`.
 
+:mod:`sklearn.isotonic`
+.......................
+
+- |Enhancement| :meth:`IsotonicRegression.fit`, :meth:`IsotonicRegression.transform`
+  and :meth:`IsotonicRegression.predict` now accepts 2darray with 1 feature as
+  input array.
+  :issue:`15012`
```

Comment:
remove this line please, we only mention PRs now

Reply:
done